accumulo/src/test/scala/geotrellis/spark/io/accumulo/AccumuloAttributeStoreSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Azavea
+ * Copyright 2017 Azavea
```

Comment:
Sounds like [here](https://github.com/locationtech/geotrellis/blob/master/build.sbt#L71-L72) changes should be done too (mb in a sep PR though)